converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java

```diff
@@ -308,6 +310,102 @@ private void createTransformation(final Element rules, final Mapping mapping) th
 		processTransformationComponentFunction(transformationComponent, mapping, mappingInputsVariablesMap, rules);
 	}
 
+	private void createLookupTable(final Element maps, final Component transformationComponent) throws DMPConverterException {
+
+		if (transformationComponent == null)
+			return;
+
+		final Function transformationFunction = transformationComponent.getFunction();
+
+		if (transformationFunction != null && transformationFunction.getFunctionType() == FunctionType.Transformation) {
```

Comment:
I'm not sure, however, I think that's better to make use of equals here instead of '==', i.e., 
````
FunctionType.Transformation.equals(transformationFunction.getFunctionType())
````